### ISSUE TITLE
Copy/Paste not working on text

### ISSUE THREAD

**Issue by mdtusz** [FormidableLabs/spectacle-editor#148]
Reproduction:

1. Create text element and ensure selection.
2. ⌘ + C and ⌘ + V, and menubar clicks don't create copies.

**Comment by mdaxtman** [comment]
@mdtusz for clarification, is the desired behavior that the element itself is copied and pasted or that the text content within the element is copied to the clipboard, and that it is overwritten when triggering `paste` ?

**Comment by mdtusz** [comment]
Ah I meant to copy/paste the _element_, not the text. I can't recall if copy paste on the text itself works - I'd have to check again.

**Comment by mdaxtman** [comment]
yes, it does on the text itself during edit mode. Do we want this behavior on other element types as well? I would assume so, right? :-)

**Comment by mdtusz** [comment]
Yep - I was initially meaning to create a duplicate of the text _node_ (if I may call it that). If adding multiple "pieces" of text, it would be nice to be able to duplicate so that the styles don't need to be re-selected each time (or alternatively, styles should be persistent?).

**Comment by mdaxtman** [comment]
okay, so as I understand it, we want a feature that will upon ⌘ + C copy the selected element, and ⌘ + V paste it onto the current slide and include all of the styles that were present on the copied element. I can add that for all of the element types if we wish, but I'll just start with text for now.

**Comment by rgerstenberger** [comment]
@mdtusz @mdaxtman I don't think we should support copy/paste at the element level right now. I'd rather we focus on other tasks first.

**Comment by rgerstenberger** [comment]
We'll keep this ticket open until we build this feature is supported.

**Comment by mdtusz** [comment]
Once I've got more free time perhaps I'll take a gander and make a PR!

**Comment by coopy** [comment]
@mdtusz: I've invited you to be a collaborator on the project, so feel free to open PRs any time. Please assign the issue to yourself first, so nobody else grabs it.
